ui/app/AppLayouts/Profile/views/ContactsView.qml

```diff
@@ -113,34 +117,59 @@ Item {
-                height: parent.height
-
-                ContactsListPanel {
-                    id: contactListView
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+
+                ColumnLayout {
                     anchors.fill: parent
-                    contactsModel: root.contactsStore.myContactsModel
-                    clip: true
-                    hideBlocked: true
-                    searchString: searchBox.text
-                    showSendMessageButton: true
-
-                    onContactClicked: {
-                        root.contactsStore.joinPrivateChat(contact.pubKey)
-                    }
 
-                    onOpenProfilePopup: {
-                        Global.openProfilePopup(contact.pubKey)
+                    ContactsListPanel {
+                        Layout.fillWidth: true
+                        Layout.preferredHeight: parent.height * 0.5
+                        contactsModel: root.contactsStore.myContactsModel
+                        clip: true
+                        title: qsTr("Identity Verified Contacts")
+                        searchString: searchBox.text
+                        panelUsage: Constants.contactsPanelUsage.verifiedMutualContacts
+
+                        onOpenProfilePopup: {
+                            Global.openProfilePopup(publicKey)
+                        }
+
+                        onSendMessageActionTriggered: {
+                            root.contactsStore.joinPrivateChat(publicKey)
+                        }
+
+                        onOpenChangeNicknamePopup: {
+                            Global.openProfilePopup(publicKey, null, true)
+                        }
                     }
 
-                    onSendMessageActionTriggered: {
-                        root.contactsStore.joinPrivateChat(contact.pubKey)
+                    ContactsListPanel {
+                        Layout.fillWidth: true
+                        Layout.preferredHeight: parent.height * 0.5
+                        contactsModel: root.contactsStore.myContactsModel
+                        clip: true
+                        title: qsTr("Contacts")
+                        searchString: searchBox.text
+                        panelUsage: Constants.contactsPanelUsage.mutualContacts
+
+                        onOpenProfilePopup: {
+                            Global.openProfilePopup(publicKey)
+                        }
+
+                        onSendMessageActionTriggered: {
+                            root.contactsStore.joinPrivateChat(publicKey)
+                        }
+
+                        onOpenChangeNicknamePopup: {
+                            Global.openProfilePopup(publicKey, null, true)
+                        }
                     }
-                    onOpenChangeNicknamePopup: {
-                        Global.openProfilePopup(contact.pubKey, null, true)
+
+                    Item {
```

Comment:
What is this Item for?

Reply:
@jrainville at the end this may be redone by rabbit cohort, or maybe even created as `StatusQ` component, but so far since I used Layouts here and split the screen on 50:50 (cause all screens, except the "Blocked Contacts", by design display 2 models) in case one of 2 model has no items I added this spacer to "fill height" and keep either first or second model at the top of the page.